File: ASSIGNMENT 01.py
cp=int(input("Enter Cost Price:"))
sp=int(input("Enter Selling Price:"))
if(sp-cp>0):
    print("Your Profit is:",sp-cp)
elif(sp==cp):
    print("You dont have any pofit or loss(0)")
else:
    print("Your Loss is:",cp-sp)

File: ASSIGNMENT 01.js
var cp = prompt("Enter Cost Price:");
if (cp < 0) {
    console.log("Cost Price cannot be negative.");
} else {
    var sp = prompt("Enter Selling Price:");
    if (sp - cp > 0) {
        console.log("Your Profit is:", sp - cp);
    } else if (sp == cp) {
        console.log("You don't have any profit or loss (0)");
    } else {
        console.log("Your Loss is:", cp - sp);
    }
}
